cdap-ui/server.js

```diff
@@ -64,6 +64,7 @@ function getFullURL(host) {
 }
 async function setAllowedOrigin() {
   const nodejsserver = cdapConfig['dashboard.bind.address'];
+  const whitelistedDomain = cdapConfig['dashboard.domain.name'];
```

Comment:
should this domain name actually be a list? Potentially user might want to add alias or multiple addresses?

Reply:
This is the domain that the user go in the browser, why it is a list?

Reply:
in private instance, customer might provide alias for the IP address